app/helpers/sessions_helper.rb

```diff
@@ -35,10 +35,34 @@ def logged_in?
     current_user.present?
   end
 
+  # Returns true if the given user is the current user.
+  def current_user? user
+    user == current_user
+  end
+
   # Logs out the current user.
   def log_out
     forget current_user
     session.delete :user_id
     @current_user = nil
   end
+
+  # Store the URL trying to be accessed.
+  def store_location
+    session[:forwarding_url] = request.original_url if request.get?
+  end
+  # redirect to stored location (or to the default).
+
+  def redirect_back_or default
+    redirect_to(session[:forwarding_url] || default)
+    session.delete :forwarding_url
+  end
+
+  def check_admin_not_current user
+    if current_user.admin? && !current_user?(user)
```

Comment:
```suggestion
    current_user.admin? && !current_user?(user)
```
chỉ cần trả về ntn là dc mà

Reply:
k cần if elase gì nữa. nó là kết quả của đoạn lệnh trên